README.md

```diff
@@ -229,3 +230,10 @@ Many thanks to all the [Scribe-Data contributors](https://github.com/scribe-org/
   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
   <br>
 </div>
+
+<a id="architecture-diagrams"></a>
+# Architecture Diagrams  [`⇧`](#contents)
```

Comment:
The table of contents references can get omitted as well, since the idea would be to simply add the blurb within the introductory section, at the end.

Reply:
@wkyoshida Noted with thanks